README.md

```diff
@@ -1,10 +1,217 @@
 [![Build Status](https://travis-ci.org/openmrs/openmrs-esm-patient-chart.svg?branch=master)](https://travis-ci.org/openmrs/openmrs-esm-patient-chart)
 
-Openmrs ESM Patient Chart
-=========
+# Openmrs ESM Patient Chart
 
-Patient dashboard microfrontend for the OpenMRS SPA
+## What is this?
 
-More info on the [wiki][]
+openmrs-esm-patient-chart is a patient dashboard microfrontend for the 
+OpenMRS SPA. It provides a simple dashboard with cards detailing the 
+patient's information, such as vitals, demographic and relationships.
 
-[wiki]: https://wiki.openmrs.org/display/projects/openmrs-esm-patient-chart
+## How do I install this?
+
+Requirements: Java 8, Latest NodeJS & NPM, Latest Git, Latest OpenMRS 
```

Comment:
Why is Java 8 a requirement?

Reply:
It isn't, except for OpenMRS itself. @PermissionError can correct me if I'm wrong.